### PR TITLE
User interface design

### PR DESCRIPTION
**Pull request by nhatquan160697**

### CODE REVIEW

**Comment by nhatquan160697** [comment]
ready

File: resources/views/layouts/user/header.blade.php [hunk 1]
```diff
@@ -0,0 +1,127 @@
...
+                        </ul>
+                    </li>
+                    <li><a href="about.html"> @lang('home_page.about_us') </a></li>
+                    <li><a href="contacts.html"> @lang('home_page.contact') </a></li>
+                </ul>
+                <div class="clearfix"></div>
+            </nav>
+        </div>
+    </div>
+</div>
```
Comment (tranducquoc) [inline resources/views/layouts/user/header.blade.php]: end file

File: resources/views/layouts/user/footer.blade.php [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+        <p class="pull-left">Privacy policy. (&copy;) 2014</p>
+        <p class="pull-right pay-options">
+            <a href="#"><img src="/source/assets/dest/images/pay/master.jpg" alt="" /></a>
+            <a href="#"><img src="/source/assets/dest/images/pay/pay.jpg" alt="" /></a>
+            <a href="#"><img src="/source/assets/dest/images/pay/visa.jpg" alt="" /></a>
+            <a href="#"><img src="/source/assets/dest/images/pay/paypal.jpg" alt="" /></a>
+        </p>
+        <div class="clearfix"></div>
+    </div> <!-- .container -->
+</div> <!-- .copyright -->
```
Comment (tranducquoc) [inline resources/views/layouts/user/footer.blade.php]: end file

**Comment by nhatquan160697** [comment]
ready

**Pull request merged**